system/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM solita/ubuntu-systemd
+
+ARG uid=1000
+
+# Install environment
+RUN apt-get update -y && apt-get install -y \
+	git \
+	wget \
+	python3.5 \
+	python3-pip \
+	python-setuptools \
+	python3-nacl \
+	apt-transport-https \
+	ca-certificates \
+	ssh \
+	vim \
+	mc
+RUN pip3 install -U \
+	'pip<10.0.0' \
+	setuptools
+COPY test_key.pub /root/.ssh/authorized_keys
+RUN sed -i 's/PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88
+RUN echo "deb https://repo.sovrin.org/deb xenial master" >> /etc/apt/sources.list
+RUN useradd -ms /bin/bash -l -u $uid indy
+RUN apt-get update -y && apt-get install -y indy-node sovtoken sovtokenfees
+RUN echo " " >> /etc/indy/indy_config.py
+RUN echo "NETWORK_NAME = 'sandbox'" >> /etc/indy/indy_config.py
+RUN echo " " >> /etc/indy/indy_config.py
```

Comment:
Each RUN adds a layer to docker filesystem, so often it is considered a good practice to group related commands (like different echoes into indy_config) into one RUN using `&&`.

Also it might make sense to move adding sovrin keys and repository before initial `apt-get update`, so only one update will ever be needed, this can significantly improve build time.

Reply:
1st - done. 2nd - should be tested according to discussion.